akka-docs/rst/project/migration-guide-persistence-experimental-2.3.x-2.4.x.rst

```diff
@@ -178,3 +178,19 @@ persistent actor on the sending side.
 
 Read more about at-least-once delivery in the :ref:`documentation for Scala <at-least-once-delivery>` and 
 :ref:`documentation for Java <at-least-once-delivery-java>`.  
+
+Default persistence plugins
+===========================
+Previously default ``akka.persistence.journal.plugin`` was set to the LevelDB journal ``akka.persistence.journal.leveldb``.
+Now persistence extension will instead use in-memory journal plugin ``akka.persistence.journal.inmem`` as a default.
+This change was needed to decouple persistence extension from the the LevelDB dependency.
+Please see persistence extension ``reference.conf`` for details. 
+
+Converted LevelDB to an optional dependency
+===========================================
+Persistence extension uses LevelDB based plugins for own development and keeps related code in the published jar.
+However previously LevelDB was a ``compile`` scope dependency, and now it is an ``optional;provided`` dependency.
+To continue using LevelDB based persistence plugins for development it is now required for related user projects
+to include an additional explicit dependency declaration for the LevelDB artifacts in the scope ``test``. 
```

Comment:
yes, in the context of "development":
"To continue using LevelDB based persistence plugins for development"